layers/+lang/typescript/packages.el

```diff
@@ -19,6 +20,17 @@
         web-mode
         ))
 
+(unless (or (configuration-layer/layer-usedp 'javascript)
+            (configuration-layer/layer-usedp 'html))
+  (defun typescript/init-add-node-modules-path ()
+    (use-package add-node-modules-path :defer t)))
```

Comment:
The first package that gets loaded gets the init. I had to write it this way when I was testing different combinations of these layers in different orders in `dotspacemacs-configuration-layers`.